pkg/ebpf/netlink/conntracker.go

```diff
@@ -333,6 +333,16 @@ func (ctr *realConntracker) compact() {
 }
 
 func isNAT(c ct.Con) bool {
+	if c.Origin == nil ||
+		c.Reply == nil ||
+		c.Origin.Proto == nil ||
```

Comment:
Weren't we originally checking the Proto? Is that code gone?

Reply:
Before we merged #4244 , ct.Con was a flat map. I think the way go returns default values was avoiding the problem (because we would just compare nil to nil).

here is the pre-refactor code:

https://github.com/DataDog/datadog-agent/blob/6.14.0/pkg/ebpf/netlink/conntracker.go#L313-L335